lmms_eval/tasks/voicebench/utils.py

```
@@ -0,0 +1,1547 @@
+    def majority_vote(scores):
+        scores = [item.lower() for item in scores]
+        final_answer = max(set(scores), key=scores.count)
+
+        # Convert the final answer to True for 'Yes' and False for 'No'
+        return True if final_answer == 'yes' else False
+    """
+
+    parsed_preds = []
+    pedant_scores = []
+    gpt_scores = []
+    combined_scores = []
```

Ruff (0.12.2)

Local variable `combined_scores` is assigned to but never used

Remove assignment to unused variable `combined_scores`

(F841)